exist-core/src/main/java/org/exist/xquery/pragmas/ForceIndexUse.java

```diff
@@ -27,13 +27,12 @@
 import org.exist.xquery.value.Sequence;
 
 public class ForceIndexUse extends Pragma {
-	
+    public static final String LOCAL_NAME = "force-index-use";
```

Comment:
See my comment above about why renaming this to `LOCAL_NAME` is problematic and needs to be reversed.